### PR TITLE
Fixed BuckleScript warnings in `Rx_Operators.re`

### PR DESCRIPTION
**Pull request by idkjs**
Im using `bs-platform@7.0.2-dev1` so getting bunch of warnings. Just wanted to pull the one file because I made some other changes in `package.json` for running. Thoughts?

### CODE REVIEW

**Comment by ambientlight** [comment]
yeah, I took definitions originally from https://github.com/mistyharsh/bs-rx/blob/master/src/Operators.ml which bind with `= ""` implicit names.

refmt makes sense to me too

File: package.json [hunk 1]
```diff
@@ -3,11 +3,11 @@
     "version": "0.1.0",
     "description": "buckleScript bindings for RxJs v7",
     "scripts": {
```
Comment (ambientlight) [inline package.json:5]: Do you think it is better to bind to global modules? This way we are making presumptions that should exist, if we are binding to local, guys would be able to run those just by installing repo dependencies

**Review by ambientlight — CHANGES_REQUESTED** [review]

**Comment by idkjs** [comment]
You should consider moving to bs-platform@7. `bs.deriving abstract` wont be necessary anymore like here:
https://github.com/ambientlight/bs-rx/blob/d5669781897144ca331ab0c294e096c48144b336/src/internal/testing/Rx_TestMessage.re#L1-L6

File: package.json (hunk 1 above)
Comment (idkjs) [reply package.json:5]: You mean for `bs-platform`? I have rarely if ever seen using bsb locally. I used to see it in the early days. I would say use global bsb based on convention.

**Comment by ambientlight** [comment]
@idkjs: the reason why I am against doing this now as many folks will have to update to bs-platform 7, I am unaware if there is any reasons not to, but it feels better if we preserve backward compatibility for now, your thoughts?

**Comment by idkjs** [comment]
Apparently there is a way to do that. @fhammerschmidt has done it on some project. Compiled in 7, released in 5 or this reason. I don't now how he did though.

**Comment by fhammerschmidt** [comment]
Not really. I only mentioned in my blog bost that we tried to compile our codebase with v7 which mostly worked out fine. We only had to ship at that time and kept 5.2.0 used in the master branch of our current app.

**Comment by idkjs** [comment]
@fhammerschmidt so we can keep bsp5 as a `peer-dev` but work in `bsp7` and the bindings should still work?

**Comment by fhammerschmidt** [comment]
Only if you write bindings the v5 way which work in both versions.

**Comment by idkjs** [comment]
Ok. Thanks for that detail. I'm for having them use bsp7 since this a new project and that's the default.

File: package.json (hunk 1 above)
Comment (ambientlight) [reply package.json:5]: unaware of such convention, alright then

**Comment by ambientlight** [comment]
@fhammerschmidt: thanks for the feedback, and @idkjs many thanks for the contribution!

**Pull request merged**